Lib/fontTools/feaLib/ast.py

```diff
@@ -770,6 +903,14 @@ def asFea(self, indent=""):
 
 
 class LigatureSubstStatement(Statement):
+    """A chained contextual substitution statement.
+
+    ``prefix``, ``glyphs``, and ``suffix`` should be lists of
+    `glyph-containing objects`_; ``replacement`` should be a single
+    `glyph-containing object`_.
+
+    If ``forceChain`` is True, this is expressed as a chaining rule
```

Comment:
I think `True` should be `true`: any true value works, not just `True`. Ditto in `MultipleSubstStatement`.

Reply:
yeah, but that should stay an implementation detail. If we were to add type hints to this it would be a `bool`